op-node/rollup/derive/ecotone_upgrade_transactions.go

```diff
@@ -42,7 +42,7 @@ var (
 )
 
 func EcotoneNetworkUpgradeTransactions() ([]hexutil.Bytes, error) {
-	upgradeTxns := make([]hexutil.Bytes, 0, 5)
```

Comment:
Why did this get reduced? By my count there are 6 transactions being added so we should start with a capacity of 6.

Reply:
I checked the history, it seems the [initial](cceb207a16) commit specified `5` and never changed afterwards.

Reply:
oh my bad - I read the diff backwards.  Sorry.

Reply:
never mind:)